test/parallel/test-vm-cached-data.js

```diff
@@ -41,12 +41,14 @@ function testProduceConsume() {
 
   const data = produce(source);
 
-  // It should consume code cache
-  const script = new vm.Script(source, {
-    cachedData: data
-  });
-  assert(!script.cachedDataRejected);
-  assert.strictEqual(script.runInThisContext()(), 'original');
+  for (const parsedData of common.getArrayBufferViews(data)) {
+    // It should consume code cache 
+    const script = new vm.Script(source, {
+      cachedData: parsedData
```

Comment:
Nit: I'd prefer to just rename `parsedData` -> `cachedData` and the this line will become just
```suggestion
      cachedData
```